src/components/ErrorBoundary/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { GlobalErrorBoundary } from '@components/ErrorBoundary/ErrorBoundary';
```

Comment:
> I wonder why you didn't choose to use barrel files to avoid Name/Name notation

@kamil-borecki-cc Honestly I just don't really like them. I like to just take a glance at a filename in a tab in my IDE and see where I am.